common/persistence/nosql/nosqlExecutionStore.go

```diff
@@ -472,7 +472,7 @@ func (d *nosqlExecutionStore) ConflictResolveWorkflowExecution(
 		ctx, currentWorkflowWriteReq,
 		mutateExecution, insertExecution, resetExecution,
 		nosqlTransferTasks, nosqlCrossClusterTasks, nosqlReplicationTasks, nosqlTimerTasks,
-		shardCondition)
+		shardCondition, 0)
```

Comment:
TTL should be also set for `ConflictResolveWorkflowExecution`

Reply:
0 means delete current TTL. https://docs.datastax.com/en/cql-oss/3.3/cql/cql_using/useExpireExample.html